site/docs/hosting/zeabur-nodejs.md

```diff
@@ -0,0 +1,133 @@
+---
+prev: false
+next: false
+---
+
+# Hosting: Zeabur (Node.js)
+
+[Zeabur](https://zeabur.com) is a platform that allows you to deploy your full-stack applications with ease. It supports various programming languages and frameworks, including Node.js and grammY.
+
+This tutorial will guide you how to deploy your grammY bots with Node.js to [Zeabur](https://zeabur.com/).
+
+::: tip Looking for the Deno Version?
+This tutorial explains how to deploy a Telegram bot to Zeabur using Node.js.
+If you're looking for the Deno version, please check out [this tutorial](./zeabur-deno) instead.
+:::
+
+
+## Prerequisites
+
+To follow along, you need to have a [Github](https://github.com) and [Zeabur](https://zeabur.com/) account.
+
+### Method 1: Create a New Project From Scratch
+
+Initialize your project and install some necessary dependencies:
+
+```sh
+# Initialize the project.
+mkdir grammy-bot
+cd grammy-bot
+pnpm init -y
+
+# Install main dependencies.
+pnpm install grammy
+
+# Install development dependencies.
+pnpm install -D nodemon
+```
+
+Then, cd into `src/`, and create a file named `bot.js`. 
```

Comment:
I actually liked it that you directly illustrated a TypeScript setup. Most people will want to do this anyway and the others (really should switch but if they don't want to then they) can just skip reading that part. Having a slightly longer page is better than leaving it to the reader to figure out how to do the setup themselves, what do you think?

Reply:
I also think using ts will be better for this project, I need to modify the template on Zeabur first and then updates the docs. Thanks for advice